src/calendar/Calendar.ts

```diff
@@ -90,6 +91,14 @@ export default class Calendar extends CalendarBase<CalendarProperties> {
 		return lastDate.getDate();
 	}
 
+	private _getMonths() {
+		const messages = this.localizeBundle(commonBundle);
```

Comment:
Can this call (and the one below in `_getWeekdays`) be moved to a member variable so it doesn't need to be called every render? This may not be possible depending on how components are notified of i18n data changes...

Reply:
I think it's already cached in the mixin. I'm going to pull Matt W in.

Reply:
After they've been loaded, messages are cached, so subsequent calls to `localizeBundle` will immediately return the cached messages without requiring an additional resource request. That said, with a future release message loading will be [handled by the bundles themselves](https://github.com/dojo/i18n/issues/106), so it might be best to make that request once and then pass in the result to private methods like `_getMonths` to avoid making multiple requests for the same unloaded bundle.